### ISSUE TITLE
Java compatibility set to Java 17 on Android with 0.54.0

### ISSUE THREAD

**Issue by akvus** [liodali/osm_flutter#413]
What is the reason to specify compatibility with Java 17 for Android? 

The package compiles without it, but specifying the compatibility causes issues for the clients using different versions of java. 

```
 ':flutter_osm_plugin:compileDebugJavaWithJavac'. > error: invalid source release: 17
```

Please see https://github.com/liodali/osm_flutter/commit/a24aac1633954374be4ab9a26e83543fa16c70a8#diff-197b190e4a3512994d2cebed8aff5479ff88e136b8cc7a4b148ec9c3945bd65aR51

**Comment by liodali** [comment]
I added desaguring and latest koltin version 1.8.20 force me to use jdk 17
it doesnt heart just change app configuration

**Comment by liodali** [comment]
the issue was the gradle-8.0 in gradle-wrapper.properties force me to use jdk 17 
i will update the configuration in next version

**Comment by liodali** [comment]
we published new version `0.54.2`

**Comment by akvus** [comment]
Thank you for the support @liodali
